src/com/compoundtheory/intellij/tmux/config/TmuxAppSettings.java

```diff
@@ -34,7 +34,9 @@
 )
 public class TmuxAppSettings implements PersistentStateComponent<TmuxAppSettings>
 {
-	public String TMUX_BINARY_PATH = "tmux";
+	public String TMUX_OS = System.getProperty("os.name");
+
+    public String TMUX_BINARY_PATH = (TMUX_OS.startsWith("Mac") ? "/opt/local/bin/tmux" : "tmux");
```

Comment:
This fails on systems with homebrew based installation or for that matter any other installation tool

Reply:
That might be the case (I can't really comment on the specifics of homebrew or macports etc), but if you have  the tmux binary in a different location, there's always the option of setting the path to your binary in IntelliJ Settings -> Other Settings -> Tmux.

If you have a better suggestion of how to determine the "ideal" pre-determined path on the different platforms, we'd appreciate a pull request.